app/grants/templates/grants/landing/round_by_type.html

```diff
@@ -0,0 +1,34 @@
+{% load humanize i18n grants_extra %}
+{% for active_round in active_rounds %}
+<div class="col-12 col-lg-6">
+  <a
+    href="/grants/explorer/?sub_round_slug={{ active_round.sub_round_slug }}&round_num={{ active_round.round_num }}&customer_name={{ active_round.customer_name }}"
+    class="active-clr-banner d-block text-decoration-none position-relative mt-3"
+  >
+    <div class="position-absolute w-100 h-100" style="
+        top: 0;
+        right: 0;
+        {% if active_round.logo %}
+          background:  url('{{ active_round.logo.url }}') no-repeat center;
+        {% endif %}
+      "
+    >
+    </div>
```

Comment:
are these inlined for a reason or can they go into a stylesheet?

Reply:
I don't think it would be possible to throw this into a stylesheet as we'd be relying on the `active_round.logo.url` which is dynamic right ?